src/q5go.pro

```diff
@@ -167,7 +167,7 @@ TRANSLATIONS	      = qgo_cz.ts \
 macx :{
     LIBS += -framework CoreFoundation
 }
-
+win32-msvc : CONFIG += c++17
```

Comment:
Where do we require C++17?

Reply:
https://github.com/bernds/q5Go/blob/81651b8662cc9a16962aa9fdb476e51e7233e261/src/setting.cpp#L497

MSVC2017 reported /std:c++17 flag is needed for this line.